test/unit/footprints.js

```diff
@@ -148,9 +148,9 @@ describe('promicro', function() {
         (fp_line (start 15.24 -8.89) (end -17.78 -8.89) (layer F.SilkS) (width 0.15))
         (fp_line (start -17.78 -8.89) (end -17.78 8.89) (layer F.SilkS) (width 0.15))
 
-        (fp_line (start -15.24 -6.35) (end -12.70 -6.35) (layer F.SilkS) (width 0.15))
-        (fp_line (start -15.24 -6.35) (end -15.24 -8.89) (layer F.SilkS) (width 0.15))
+        (fp_line (start -12.70 -6.35) (end -15.24 -6.35) (layer F.SilkS) (width 0.15))
```

Comment:
@mrzealot note that this describes the exact same geometry as removed line 151, merely `start` and `end` are swapped.